Casks/balenaetcher.rb

```diff
@@ -26,7 +26,6 @@
   ]
 
   caveats <<~EOS
-    Support for future updates of #{token} has been discontinued in homebrew-cask
-    due to extremely frequent stable releases.
+    #{token} is no longer updated in homebrew-cask due to multiple stable releases per day.
```

Comment:
The releases aren't the issue. Our time is.

```suggestion
    #{token} is no longer updated in homebrew-cask due to the maintenance demand of multiple stable releases per day.
```